Lib/fontTools/otlLib/builder.py

```diff
@@ -695,14 +1270,29 @@ def buildBaseArray(bases, numMarkClasses, glyphMap):
 
 
 def buildBaseRecord(anchors):
-    """[ot.Anchor, ot.Anchor, ...] --> ot.BaseRecord"""
+    # [otTables.Anchor, otTables.Anchor, ...] --> otTables.BaseRecord
+    # (Not docstringed because it's only needed when building BaseArray,
+    # at which point you should be using the above.)
```

Comment:
this comment is not necessary, we just stick to the implicit rule that if something hasn't a `"""` docstring it isn't in the publicly documented API